packages/firebase_ui_localizations/lib/src/lang/he.dart

```diff
@@ -1,132 +1,121 @@
-  String get confirmPasswordInputLabel => "וודא סיסמה"; // "Confirm password";
+  String get confirmPasswordInputLabel => "וודא סיסמה";
 
   @override
-  String get confirmPasswordIsRequiredErrorText =>
-      "וודא את הסיסמה שלך"; //"Confirm your password";
+  String get confirmPasswordIsRequiredErrorText => "וודא את הסיסמה שלך";
 
   @override
-  String get continueText => "המשך"; //"Continue";
+  String get continueText => "המשך";
 
   @override
-  String get countryCode => "קידומת"; //"Code";
+  String get countryCode => "קידומת";
 
   @override
   String get credentialAlreadyInUseErrorText =>
-      "ספק זה משויך לחשבון משתמש אחר."; //"This provider is associated with a different user account.";
+      "ספק זה משויך לחשבון משתמש אחר.";
 
   @override
-  String get deleteAccount => "מחיקת חשבון"; //"Delete account";
+  String get deleteAccount => "מחיקת חשבון";
 
   @override
   String get differentMethodsSignInTitleText =>
-      "השתמש באחת מהשיטות הבאות כדי להיכנס"; //"Use one of the following methods to sign in";
+      "השתמש באחת מהשיטות הבאות כדי להיכנס";
 
   @override
-  String get disable => "השבת"; //"Disable";
+  String get disable => "השבת";
 
   @override
   String get eastInitialLabel => "E";
 
   @override
-  String get emailInputLabel => "דואר אלקטרוני"; //"Email";
+  String get emailInputLabel => "דואר אלקטרוני";
 
   @override
-  String get emailIsRequiredErrorText =>
-      "דואר אלקטרוני נדרש"; //"Email is required";
+  String get emailIsRequiredErrorText => "דואר אלקטרוני נדרש";
 
   @override
-  String get emailLinkSignInButtonLabel =>
-      "כניסה באמצעות קישור קסם"; //"Sign in with magic link";
+  String get emailLinkSignInButtonLabel => "כניסה באמצעות קישור קסם";
 
   @override
-  String get emailTakenErrorText =>
-      """"חשבון עם דוא"ל זה כבר קיים"""; //"An account with this email already exists";
+  String get emailTakenErrorText => "חשבון עם דואייל זה כבר קיים";
 
   @override
-  String get enable => "הפעל"; //"Enable";
+  String get enable => "הפעל";
 
   @override
-  String get enableMoreSignInMethods =>
-      "הפעל שיטות כניסה נוספות"; //"Enable more sign-in methods";
+  String get enableMoreSignInMethods => "הפעל שיטות כניסה נוספות";
 
   @override
-  String get enterSMSCodeText => "הזן קוד SMS"; //"Enter SMS code";
+  String get enterSMSCodeText => "הזן קוד SMS";
 
   @override
-  String get findProviderForEmailTitleText =>
-      """הזן את הדוא"ל שלך כדי להמשיך"""; //"Enter your email to continue";
+  String get findProviderForEmailTitleText => "הזן את הדואייל שלך כדי להמשיך";
 
   @override
-  String get forgotPasswordButtonLabel =>
-      "שכחת את הסיסמה?"; //"Forgotten password?";
+  String get forgotPasswordButtonLabel => "שכחת את הסיסמה?";
 
   @override
   String get forgotPasswordHintText =>
-      """הזן את הדוא"ל שלך ואנו נשלח לך קישור לאיפוס הסיסמה"""; //"Provide your email and we will send you a link to reset your password";
+      "הזן את הדואייל שלך ואנו נשלח לך קישור לאיפוס הסיסמה";
```

Comment:
That's incorrect, you should put double quotes (") instead of יי
```suggestion
"""הזן את הדוא"ל שלך ואנו נשלח לך קישור לאיפוס הסיסמה""";
```